packages/gatsby-cli/src/reporter/loggers/ink/components/error.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react"
+import path from "path"
+import { Color, Box } from "ink"
+import { get } from "lodash"
+
+interface IFileProps {
+  filePath: string
+  location: string
+}
+const File = ({ filePath, location }: IFileProps): JSX.Element => {
```

Comment:
```suggestion
const File: FunctionComponent<IIFileProps> = ({ filePath, location }) => {
```

Reply:
I like the other approach better

Reply:
I want to unify which statement should be used, `React.FC` or `React.FunctionComponent`.
https://github.com/gatsbyjs/gatsby/pull/22084#discussion_r390042955

Reply:
@blainekasten Using `FunctionComponent` will give you correcter types (it includes `children` for you) and will make sure you can't invoke `File` as a normal function, but can only use it as a component like `<File />`.

@cola@ I always use `FunctionComponent` so it's a lot clearer what it means

Reply:
I have to agree with @MichaelDeBoey . `FunctionComponent` is the right way to do this.